src/common/database.cpp

```diff
@@ -150,7 +150,11 @@ void YamlDatabase::parse( const YAML::Node& rootNode ){
 		const char* fileName = this->currentFile.c_str();
 
 		for( const YAML::Node &node : bodyNode ){
-			count += this->parseBodyNode( node );
+			if( this->nodeExists( node, "Remove" ) ){
```

Comment:
The doc is missing for the user to understand how to use it (maybe add it for each yml doc)